Tasks/Kubernetes/task.json

```diff
@@ -0,0 +1,207 @@
+        },
+        {
+            "name": "forceUpdate",
+            "type": "boolean",
+            "label": "Force Update secret",
+            "defaultValue": "true",
+            "helpMarkDown": "Recreate secret.",
+            "groupName": "registry"
+        },
+        {
+            "name": "command",
+            "type": "pickList",
+            "label": "Command",
+            "defaultValue": "apply",
+            "required": true,
+            "options": {
+                "apply": "apply",
+                "create": "create",
+                "delete": "delete",
+                "exec": "exec",
+                "expose": "expose",
+                "get": "get",
+                "logs": "logs",
+                "run": "run",
+                "set": "set",
+                "top": "top"
+            },
+            "helpMarkDown": "Select a kubectl command.",
+            "groupName": "commands",
+            "properties": {
+                "EditableOptions": "True"
+            }
+        },
+        {
+            "name": "useConfigurationFile",
+            "type": "boolean",
+            "label": "Use Configuration files",
+            "defaultValue": "false",
+            "helpMarkDown": "Use configuration file for the command.",
+            "groupName": "commands"
+        },
+        {
+            "name": "configuration",
+            "type": "filePath",
+            "label": "Configuration File",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Path to the Configuration file or directory.",
+            "visibleRule": "useConfigurationFile = true",
+            "groupName": "commands"
+        },
+        {
+            "name": "arguments",
+            "type": "multiLine",
+            "properties": {
+                "resizable": "true",
+                "rows": "2"
+            },
+            "label": "Arguments",
+            "helpMarkDown": "Command arguments",
+            "groupName": "commands"
+        },
+        {
+            "name": "cwd",
+            "type": "filePath",
+            "label": "Working Directory",
+            "defaultValue": "$(System.DefaultWorkingDirectory)",
+            "helpMarkDown": "Working directory for the Kubectl command.",
+            "groupName": "commands"
+        },
+        {
+            "name": "outputFormat",
+            "type": "pickList",
+            "label": "Output format",
+            "defaultValue": "json",
+            "required": false,
+            "options": {
+                "json": "json",
+                "yaml": "yaml"
+            },
+            "helpMarkDown": "output file format.",
+            "groupName": "output"
+        },
+        {
+            "name": "kubectlOutput",
+            "type": "string",
+            "label": "Kubectl output",
+            "required": false,
+            "defaultValue": "",
+            "groupName": "output",
+            "helpMarkDown": "Name of the variable."
```

Comment:
I am yet to receive the helptext from Atul. Will update all the help text once i get from Atul.